projects/go-lib/src/lib/components/go-badge/go-badge.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, HostBinding, Input } from '@angular/core';
+
+@Component({
+  selector: 'go-badge',
+  templateUrl: './go-badge.component.html',
+  styleUrls: ['./go-badge.component.scss']
+})
+export class GoBadgeComponent {
+  @Input() badgeData: string;
+  @Input() badgeColor: string = 'neutral';
+  @Input() displayData: boolean = true;
```

Comment:
Should we set the `displayData` to false by default?

Reply:
I think this should be true by default. The reason I say this is because most times you are adding a badge you would expect whatever data you are adding to that badge to show up. There are very few instances that I can think of where you would just want the badge without the data showing up and all of them are pretty conditional.